cognite/neat/_graph/extractors/_classic_cdf/_base.py

```diff
@@ -263,7 +263,11 @@ def _as_object(self, raw: Any, key: str) -> Literal | URIRef:
             "created_time",
             "last_updated_time",
         } and isinstance(raw, int):
-            return Literal(datetime.fromtimestamp(raw / 1000, timezone.utc), datatype=XSD.dateTime)
+            try:
+                return Literal(datetime.fromtimestamp(raw / 1000, timezone.utc), datatype=XSD.dateTime)
+            except (OSError, ValueError) as e:
+                warnings.warn(NeatValueWarning(f"Failed to convert timestamp {raw} to datetime: {e!s}"), stacklevel=2)
+                return Literal(raw)
```

Comment:
Maybe nitpick (I don't know the full context): 
Is it safe to return as a literal even if it fails to convert? Sounds a bit risky

Reply:
We check for integer above, so it will be just one integer triple.